mlserver/grpc/converters.py

```diff
@@ -310,6 +324,10 @@ def from_types(cls, type_object: types.InferenceResponse) -> pb.ModelInferRespon
             ],
         )
 
+        if use_raw:
+            # If using raw outputs, ensure it's set on the final object
+            model_infer_response.raw_output_contents.extend(raw)
```

Comment:
The assumption here is that raw contents is always the last field in the grpc message, which is fine with the current spec. I just wonder is the spec guarantees this is always the case? it makes sense to be always the case but just checking.

Reply:
oh yes I just misread the code. it is fine.